gulpfile.js

```diff
@@ -232,4 +273,37 @@ gulp.task("release", ["build"], function () {
                 fs.writeFileSync(path.join(__dirname, fileName), fs.readFileSync(path.join(backupFolder, fileName)));
             });
         });
-});
\ No newline at end of file
+});
+
+gulp.task('add-i18n', function () {
```

Comment:
@SounD120 Could you please add comments for each of the new gulp tasks with short description of what it does?

Reply:
Yes, sure